app/views/deployments.html

```diff
@@ -96,8 +98,66 @@
                   </tr>
                 </tbody>
               </table>
-              <div ng-if="(unfilteredReplicationControllers | hashSize) > 0">
-                <h3>Other replication controllers</h3>
+              <div ng-if="k8sDeployments | hashSize">
+                <h3>Deployments</h3>
+                <table class="table table-bordered table-hover table-mobile">
+                  <thead>
+                    <tr>
+                      <th>Name</th>
+                      <th>Last Version</th>
+                      <th>Replicas</th>
+                      <th>Created</th>
+                      <th>Strategy</th>
+                    </tr>
+                  </thead>
+                  <tbody ng-repeat="k8sDeployment in k8sDeployments | orderObjectsByDate : true">
+                    <tr>
+                      <td data-title="Name">
+                        <a ng-href="{{k8sDeployment | navigateResourceURL}}">{{k8sDeployment.metadata.name}}</a>
+                      </td>
+                      <td data-title="Last Version">
+                        #{{k8sDeployment | annotation : 'deployment.kubernetes.io/revision'}}
```

Comment:
we usually try to fallback cleanly in some way of there if an annotation is missing, for now could just say Unknown.  once we can map the RSes in this should be the link to the latest RS and can be the name of that RS if the annotation is missing like we do for DC->RC